how_to_build.md

```diff
@@ -9,17 +9,22 @@ At the time of writing the build is only confirmed to work on Windows using the
 
 ## Prerequisites
 
-1. Ensure you have .NET framework 4.6.1 or later installed.
+The build requires that a few pieces of software be installed on the host computer. We're somewhat aggressive about adoptiong new language features and the like, so rather than specifying exactly which versions are required, we'll tend toward
+"latest" or "at least" forms of guidance. If it seems you have an incompatible version of the software, prefer to upgrade rather than downgrade.
 
-1. Ensure you have version 15.3 or later of either Visual Studio 2017 or MSBuild installed.
+Ensure that the following are installed:
 
-1. Ensure you have .NET Core SDK 2.0.2 or later installed
+1. .NET framework 4.6.1 or later
+
+1. a very recent version of Visual Studio 2017 or MSBuild (currently this means 15.7 or later)
+
+1. a very recent version of the .NET Core SDK (currently this means 2.1.401 or later)

```

Comment:
I don't think we need a version this high. It works fine with 2.1.4 for instance.